### PR TITLE
minor edit in the document

### PR DESCRIPTION
**Pull request by joinimran**
Initial steps revision in the document as few terms have changed.

Problem: https://github.com/MicrosoftDocs/microsoft-365-docs/issues/1076

### CODE REVIEW

**Comment by joinimran** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

@mypil FYI Please

File: microsoft-365/security/office-365-security/detect-and-remediate-illicit-consent-grants.md [hunk 1]
```diff
@@ -31,11 +31,13 @@ You need to search the Office 365 **audit log** to find signs, also called Indic
 
 1. Open the **Security and Compliance Center** in your Office 365 tenant.
 
-2. Navigate to the **Search & investigation** node and select **audit log** search.
+2. Navigate to the **Search** and select **Audit log search**.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/detect-and-remediate-illicit-consent-grants.md]:
```suggestion
2. Navigate to **Search** and select **Audit log search**.
```

File: microsoft-365/security/office-365-security/detect-and-remediate-illicit-consent-grants.md [hunk 2]
```diff
@@ -31,11 +31,13 @@ You need to search the Office 365 **audit log** to find signs, also called Indic
...
 
-3. Create a search (all activities and all users) and filter the results for Consent to application, and Add OAuth2PermissionGrant.
+3. Search (all activities and all users) and enter the start date and end date if required and then click **Search**. 
 
-4. Examine the Extended Properties and check to see if IsAdminContent is set to True.
+4. Filter the results for Consent to application, and Add OAuth2PermissionGrant.
+
+5. Click on the result to see the details of the activity. Click **More Information** to get deatils of the activity. Check to see if IsAdminContent is set to True.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/detect-and-remediate-illicit-consent-grants.md]:
```suggestion
5. Click on the result to see the details of the activity. Click **More Information** to get details of the activity. Check to see if IsAdminContent is set to True.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Comment by joinimran** [comment]
@mypil Ready to go live.

**Comment by mypil** [comment]
@chrfox - PR has been copy edited and is ready for final review. This is based on issue #1076

Thank you.

cc: @kenwith

**Pull request merged**